app/assets/javascripts/listings/components/propertyCard.js.coffee

```diff
@@ -44,6 +44,23 @@ angular.module('dahlia.components')
             types.push(ListingDataService.reservedLabel(listing, type, 'reservedForWhoAre'))
         if types.length then types.join(' or ') else ''
 
+      @hasSaleUnitsWithoutParking = (listing) ->
+        for unitSummary in listing.unitSummaries.general || []
+          if unitSummary.minPriceWithoutParking
+            return true
+        false
+
+      @hasSaleUnitsWithParking = (listing) ->
+        for unitSummary in listing.unitSummaries.general || []
+          if unitSummary.minPriceWithParking
+            return true
+        false
+
+      @hasRangeOfPrices = (unitSummary, withParking) ->
```

Comment:
any thoughts on this as an approach rather than having duplicate functions for everything (one for with parking, one for without?